pkg/copy_backupconfiguration.go

```diff
@@ -0,0 +1,113 @@
+package pkg
+
+import (
+	"fmt"
+	"stash.appscode.dev/stash/apis/stash/v1beta1"
+	stash_v1beta1_util "stash.appscode.dev/stash/client/clientset/versioned/typed/stash/v1beta1/util"
+
+	"github.com/appscode/go/log"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kerr "k8s.io/apimachinery/pkg/api/errors"
+)
+
+func NewCmdCopyBackupConfiguration() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:               "bc",
+		Short:             `Copy BackupConfiguration`,
+		Long:              `Copy BackupConfiguration from one namespace to another namespace`,
```

Comment:
Use it in short description

Reply:
Long description should inform that it will copy respective Repository and Secret if they are not present target namespace.